cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -4803,7 +4803,7 @@ declare interface Float16 {
  * @experimental
  */
 declare interface Float16Constructor {
```

Comment:
Shouldn't this one be `Float16ArrayConstructor`?

Reply:
This looks good as is.

https://github.com/microsoft/TypeScript/blob/31afb988d1c6857ce4d1ab3b2558385e68a565a8/src/lib/es2015.iterable.d.ts#L471-L481